### ISSUE TITLE
Build Error on Batman

### ISSUE THREAD

**Issue by danleecreates** [colinmeinke/svg-tween#6]
I tried to run the build on one of the examples (Batman) and I got the following error:

> 22 error @0.0.0-semantically-released build: `npm run copy && webpack ./client.js ./client.dist.js --config ./webpack.config.babel.js && npm run clean`

**Comment by colinmeinke** [comment]
Seems to be working for me locally. What's your setup?

**Comment by nearkingdom** [comment]
Ok this is the full error:

Any idea what I should do to fix?

```
Failed at the @0.0.0-semantically-released build script 'npm run copy && webpack ./client.js ./client.dist.js --config ./webpack.config.babel.js && npm run clean'.
npm ERR! This is most likely a problem with the  package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run copy && webpack ./client.js ./client.dist.js --config ./webpack.config.babel.js && npm run clean
```

**Comment by colinmeinke** [comment]
What's your setup?

**Comment by nearkingdom** [comment]
I'm on a Mac. Not too sure what you are asking - what details do you need to know?

I cloned svg-tween-master and ran npm install

I then cd into examples/batman and did `npm install` and `npm build`

**Comment by colinmeinke** [comment]
What version of node? What version of npm?

**Comment by nearkingdom** [comment]
I'm using the following...

node 0.12.2
npm 3.3.5

**Comment by colinmeinke** [comment]
Ah that's probably the issue. If you can, upgrade to node 6. You can use https://github.com/creationix/nvm to manage node versions if you still have projects that require 0.12.

**Comment by nearkingdom** [comment]
Ok I have updated and ran `npm install`

I get the new message : `babel-loader@6.2.7 requires a peer of babel-core@^6.0.0 but none was installed.`

What is it asking me to do?

**Comment by colinmeinke** [comment]
This should have fixed that issue d72266d69c5ad2f44bc0a890b13b49ec8b04b4a5
